src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter_CollectionExpression.cs

```diff
@@ -1075,19 +1077,22 @@ private BoundExpression CreateAndPopulateList(BoundCollectionExpression node, Ty
 
                             // If collection has a struct enumerator but doesn't implement ICollection<T>
```

Comment:
Nit: 
```suggestion
                            // If collection has a struct enumerator but doesn't have a conversion to ICollection<T>
```

Reply:
Thanks. I'll hold off changing the comment for now because changes here will conflict with another PR in progress, and since I believe the existing comment captures the intent. We can update later if needed.